### PR TITLE
[WNMGDS-1914] Add a visual regression testing suite with Playwright

### PR DESCRIPTION
**Pull request by pwolfert**
## Summary

https://jira.cms.gov/browse/WNMGDS-1914

- Adds a visual regression testing suite with Playwright

## Reviewing

[Here is a link to filter out the image files in the diff](https://github.com/CMSgov/design-system/pull/2142/files?file-filters%5B%5D=.json&file-filters%5B%5D=.jsx&file-filters%5B%5D=.lock&file-filters%5B%5D=.md&file-filters%5B%5D=.ts&file-filters%5B%5D=dotfile&show-viewed-files=true)

## How to test

1. `yarn install`
2. `yarn build:storybook`
3. `yarn test:browser:docker` - Needs to run in Docker on our machines for consistency. On Jenkins the whole job runs within the same docker container, so it runs with `yarn test:browser`
    - Feedback requested: Should I remove the `:docker` sub-command for consistency and have `yarn test:browser` always run in docker? Jenkins would then just have to run playwright directly with `yarn playwright test --config tests/browser/playwright.config.ts`

## Checklist

- [x] Prefixed the PR title with the [Jira ticket number](https://jira.cms.gov/projects/WNMGDS/) as `[WNMGDS-####] Title`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,14 +16,14 @@
     "build:tokens": "yarn --cwd ./packages/design-system-tokens build themes scss && yarn --cwd ./packages/design-system-tokens build themes css-vars && yarn --cwd ./packages/design-system-tokens dist",
     "deploy-demo": "ts-node ./scripts/deployDemo.ts",
     "lint": "yarn lint:stylelint && yarn lint:eslint",
-    "lint:eslint": "npx eslint ./packages/",
-    "lint:stylelint": "npx stylelint ./packages/**/*.scss --custom-formatter=node_modules/stylelint-formatter-pretty",
+    "lint:eslint": "yarn eslint ./packages/",
+    "lint:stylelint": "yarn stylelint ./packages/**/*.scss --custom-formatter=node_modules/stylelint-formatter-pretty",
```
Comment (pwolfert) [inline package.json:20]: For some reason the `npx` command was failing in Jenkins in the new docker container, so I just changed it here.

**Review by forrestbaer — APPROVED** [review]
I would say good to remove the `:docker` command and just run it manually in CI. All looks good but I wonder how much we need the multiple browsers, I guess that remains to be seen.

Nice stuff, ran fairly quickly on my local machine but definitely slowed things down while it was running.

**Comment by pwolfert** [comment]
A point to clarify: the `test:browser:docker` command is actually _for our local development environments_ to ensure consistency, and Jenkins will run `test:browser` because Jenkins runs the whole job in the docker container. If I removed `test:browser:docker`, the yarn command would include the dockr code

**Comment by pwolfert** [comment]
I will address the `:docker` thing in a follow-up PR. Merging now so that I can put up the a11y test changes that correspond with my jenkins script changes

**Pull request merged**